scripts/run-sonata-sim.sh

```diff
@@ -0,0 +1,55 @@
+#!/bin/bash
+
+set -e
+
+if [ -z "${SONATA_SIMULATOR}" ] ; then
+	SONATA_SIMULATOR=/cheriot-tools/bin/sonata_simulator
+fi
+
+if [ -z "${SONATA_SIMULATOR_BOOT_STUB}" ] ; then
+	SONATA_SIMULATOR_BOOT_STUB=/cheriot-tools/elf/sonata_simulator_boot_stub
+fi
+
+if [ -z "$1" ] ; then
+	echo You must specify an elf file to run.
+	exit 1
+fi
+
+if [ ! -x "${SONATA_SIMULATOR}" ] ; then
+	echo Unable to locate Sonata simulator, please set SONATA_SIMULATOR to the full path of the simulator.
+	exit 2
+fi
+
+if [ ! -x "${SONATA_SIMULATOR_BOOT_STUB}" ] ; then
+	echo Unable to locate Sonata simulator boot stub, please set SONATA_SIMULATOR_BOOT_STUB to the full path of the boot stub.
+	exit 3
+fi
+
+# Remove old uart log
+UART_LOG=uart0.log
```

Comment:
Possibly make this also parametric as the above paths?

Reply:
Good idea, this is done now.